artemis-core-client/pom.xml

```diff
@@ -118,6 +118,10 @@
          <groupId>io.netty</groupId>
          <artifactId>netty-common</artifactId>
       </dependency>
+      <dependency>
+         <groupId>com.google.guava</groupId>
```

Comment:
why you need that? core-client doesn't use it directly.. only commons is using guava. I thought we should keep it that way.

Reply:
I would prefer not having more collections added directly on core-client.. having it concentrated through commons would avoid that from being added. (wouldn't compile if someone added by mistake)

Reply:
The artemis-core-client module *is* using Guava directly. If you run `mvn dependency:analyze` without my fix you'll see it complain about the undeclared dependency or you can just look in `org.apache.activemq.artemis.core.message.impl.CoreMessageObjectPools` to see where Guava is used.

Reply:
yeah.. you're right.